annotator/Dockerfile

```diff
@@ -26,7 +41,7 @@ RUN cd /usr/local/apache2/htdocs && tar -xvzf brat-v1.3_Crunchy_Frog.tar.gz && m
 COPY cgi-bin /usr/local/apache2/htdocs/brat
 RUN chmod +x /usr/local/apache2/htdocs/brat/*.sh
 
-RUN apt-get update && apt-get install -y python
+RUN #apt-get update && apt-get install -y python
```

Comment:
Uff this is quite complex compared to previous version  ... isn't there something like 
`apt-get install -y python3` that would solve this issue ... you can always run the service then login to the running container and try to install it like that. It does not work ?

Reply:
ahhh ... sorry no I see that it is 19msmt-distribution/annotator .... maybe it requires python2 right?

Reply:
Yes exactly, I tried with Python3, it didn't work. But I didn't investigate so much, it can be worth it to investigate a bit more.

Reply:
@blcham I would keep this solution, I tried to make it work with Python3, but I cannot succeed. Also, it is likely that even if we make it work with python3, the workaround will be even more complex that this PR. Also, running with python3 might introduce unexpected behavior.

Reply:
ok